tabbycat/notifications/consumers.py

```diff
@@ -91,6 +92,8 @@ def email(self, event: Dict[str, Union[str, List[int], Dict[str, Any]]]) -> None
         messages = []
         records = []
         for instance, recipient in data:
+            instance = asdict(instance)
```

Comment:
In general, I agree, but in this specific case I feel it makes sense to cast it to a dictionary.  None of `instance`'s attributes are ever accessed. It is only ever passed to other constructors, which expect dictionaries. As such, all that not casting it to a `dict` would achieve is having two instances of `asdict(instance)` instead of one as is the case now.